ccd-definition/CaseEventToComplexTypes/Response/PartAdmit/DefendantBankAccounts.json

```diff
@@ -0,0 +1,26 @@
+[
+  {
+    "ID": "BankAccount",
```

Comment:
Please could you change the  remove the package name 'response' and 'partadmit' and place it directly under CaseEventToComplexTypes. The screens/types may be shared across journeys and work streams, so it would allow to reuse where we can, else others may duplicate the type

Reply:
On second observation, this file maynot be needed because of identical file contents in ccd-definition/CaseEventToComplexTypes/DefendantBankAccounts.json